src/datepicker/datepicker.ts

```diff
@@ -203,6 +203,11 @@ export class NgbDatepicker implements OnDestroy,
    */
   @Output() navigate = new EventEmitter<NgbDatepickerNavigateEvent>();
 
+  /**
+   * An event fired when select happens.
+   */
+  @Output() onSelect = new EventEmitter<NgbDate>();
```

Comment:
Could you please change this to: 
```javascript
@Output() select = new EventEmitter<NgbDateStruct>();
```

select → to keep naming consistent
`NgbDateStruct` → as `NgbDate` is internal implementation